packages/api/src/utils/billing-types.ts

```diff
@@ -80,7 +80,9 @@ export interface SubscriptionsService {
 }
 
 export interface BillingUser {
```

Comment:
I don't think we should have the name and email required in here because not all billing things need it. A lot of times all that's needed is the ID. If we make it required, certain steps will either require passing mocked data or incur an unnecessary read from the db or session.

Consider the proposed solution I [wrote here](https://www.notion.so/bizops-can-see-name-and-email-in-stripe-com-dashboard-for-customers-9e263386e79746fabffaf38d66a0dfc6#1df8960aa93642e496c7cbd7c908816a) that @jsdevel acknowledged in notion?

Reply:
Sorry @gobengo , I hadn't added reviewers since I'd opened it up to initiate discussion with @jsdevel (I'll take another approach to this next time 😆 ).  I've since reviewed the notion doc and hopefully the changes are more in line with your planning.